DDDEastAnglia.Tests/Helpers/LoginMethods/LoginMethodIconProviderTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using DDDEastAnglia.Helpers.AppSettings;
+using DDDEastAnglia.Helpers.LoginMethods;
+using NSubstitute;
+using NUnit.Framework;
+
+namespace DDDEastAnglia.Tests.Helpers.LoginMethods
+{
+    [TestFixture]
+    public sealed class LoginMethodIconProviderTests
+    {
+        [Test]
+        public void GetIcon_ReturnsTheConfiguredIconForOAuthProvider()
+        {
+            var appSettingsProvider = Substitute.For<IAppSettingsProvider>();
+            appSettingsProvider.GetSetting(LoginMethodIconProvider.AppSettingsKeyPrefix + "test").Returns("test-icon");
```

Comment:
Furthermore, it wasn't until I actually saw it in the config file that I realised that this didn't feel right to me: appSettings is intended for configuration data; the icon associated with a login method isn't configuration data, it's just a user display thing. Enabling/disabling a login method - ok for appSettings; defining how a login method should be displayed to the user - not ok for appSettings.